Description: Don't link X libs if unneeded
Author: Ximin Luo <infinity0@debian.org>
Forwarded: TODO
---
This patch header follows DEP-3: https://dep.debian.net/deps/dep3/
--- giac.orig/src/Makefile.am
+++ giac/src/Makefile.am
@@ -51,7 +51,7 @@
 icas_SOURCES = icas.cc 
 icas_LDADD = libxcas.la $(GINSH_LIBS) $(GSL_LIBS) $(LAPACK_LIBS)
 aide_SOURCES = aide.cc
-aide_LDADD = libxcas.la $(GINSH_LIBS) $(GSL_LIBS) $(LAPACK_LIBS)
+aide_LDADD = libgiac.la $(GINSH_LIBS) $(GSL_LIBS) $(LAPACK_LIBS)
 hevea2mml_SOURCES = hevea2mml.cc lex_yy.c y_tab.c
 hevea2mml_LDADD = 
 
